pagespeed/automatic/proxy_fetch.cc

```diff
@@ -1035,7 +1035,8 @@ void ProxyFetch::ExecuteQueued() {
   bool do_flush = false;
   bool do_finish = false;
   bool done_result = false;
-  bool force_flush = false;
+  bool force_flush = network_flush_outstanding_
+      && Options()->follow_flushes();
```

Comment:
This changes the behavior with flush_html.
Also a bit weird to set network_flush_outstanding after checking flush_html  (along with || follow_flushes()) and then just ignore it.